ocs_ci/ocs/cluster.py

```diff
@@ -586,6 +693,230 @@ def get_throughput_percentage(self):
         logging.info(f"The throughput percentage of the cluster is {throughput_percentage}%")
         return throughput_percentage
 
+    def osd_health_check(self, count):
+        """
+        Osd health check based on pod count
+
+        Args:
+            count (int): Expected number of Osd pods
+
+        Raises:
+            OsdCountException: if osd pod count doesn't match
+        """
+        timeout = 10 * len(self.pods)
+        logger.info(f"Expected OSDs = {count}")
+        try:
+            assert self.POD.wait_for_resource(
+                condition='Running', selector=self.osd_selector,
+                resource_count=count, timeout=timeout, sleep=3,
+            )
+
+            actual_osds = len(pod.get_osd_pods())
+
+            assert count == actual_osds, f"Expected {count},  Got {actual_osds}"
```

Comment:
change to: "Expected OSD count {count}..."

Reply:
Done